examples/_includes/macros/tab/styles.xml.njk

```diff
@@ -1,2 +1,35 @@
 <style id="tab-content" flex="1" />
 <style id="centered" alignItems="center" justifyContent="center" />
+<style
+  id="row"
+  backgroundColor="#eee"
+  borderRadius="12"
+  flexDirection="row"
+  justifyContent="space-between"
+  marginBottom="24"
+  marginHorizontal="24"
+  padding="2"
+/>
+<style
+  id="tab"
+  alignItems="center"
+  borderColor="transparent"
+  borderRadius="8"
+  borderWidth="1"
+  flex="1"
+  padding="8"
+  justifyContent="center"
+>
+  <modifier selected="true">
+  <style
+  backgroundColor="white"
```

Comment:
is prettier causing this indentation issue?

Reply:
yes ;/